test/test_helper.rb

```diff
@@ -21,7 +21,7 @@ def add_car_to_cart
     create_cars(1)
     visit cars_path
 
-    click_button 'Drive'
+    click_link 'Add to my Garage'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.